### ISSUE TITLE
Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (19 - Representative Sampling)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#461]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

FPWD: Representative sampling is an important strategy to large and complex sites. We welcome suggestions and feedback about the best way to incorporate representative sampling into WCAG 3.0.

Representative sampling will be a highly customized process for different content providers. Sampling techniques have different issues for different types of sites. For example, a sampling process that requires manual testing on 5% of views would be a major challenge for a site with 50,000 views (2,500 manual view tests), let alone one with millions. A requirement that the content provider selects the top ten workflows to test is much more achievable. However, workflow definition may vary greatly between providers so guidance is needed on what level of detail should be included.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.
